### ISSUE TITLE
Data order

### ISSUE THREAD

**Issue by AndrewAkinson** [richsmith/sexytopo#68]
Currently the data order appears to be numerical, this is not useful, if stations are 1, 2, 3, 4 taken in that order then you return to 2 the data curretly appears as 1.1 but between station 1 and 2. It is far simpler for the surveyor to have the data appear at the end of the file. This means if legs are taken accidently they are easier to find and remove as it is just the last x stations. It is also important in post processing that the data is in order. If the distoX goes wrong or a team member changes and causes a problem (wearing metal watch or something) and the place where the happened, or at least a place where the survey was definitely correct can be identified, only data after that point has to be binned. The data order is not always the same as the cave passage order.

**Comment by CaverBruce** [comment]
Tried using ST in a simulated real survey scenario for the first time over the weekend.  And took the opportunity to start training a friend in the principles of paperless surveying (using ST v1.2.2).
Eventually we wanted to attach splay shots to a previous station, as we'd accidentally shot a survey leg before taking all splays that we wanted.  So we set the active station to the desired station and took the splays.  To our confusion the subsequent shots were not placed at the bottom of the table.  Took us some time to find them, and even then we were not sure which splay was which.  In my opinion the data table order MUST exactly match the order in which they are transferred from disto to device.  An option to sort differently would be acceptable, but original order should be the primary setting and should be always subsequently available.
Related to this, renaming a station causes that leg, and all the splays from it to be sorted to the bottom of the table.  The correct behaviour would be for none of the legs or splays to have their position changed due to any kind of edit or renaming.
